spec/requests/media/odk_response_spec.rb

```diff
@@ -35,14 +47,113 @@
       expect(FileUtils.rm(tmp_files)).to be_empty
     end
 
+    it "should safely ignore simple duplicates", database_cleaner: :truncate do
```

Comment:
did creating the request specs help reveal anythinig?

Reply:
it revealed that everything works as I would have expected, no weird obvious bugs, and no blank responses being left behind (which is great, and also prolongs the mystery). enough has changed with the new random delay that I don't want to keep throwing more bologna at the wall until we see if this one stuck.